README.md

```diff
@@ -79,198 +83,58 @@ requires the [CUDAnative.jl](https://github.com/JuliaGPU/CUDAnative.jl) package.
-   occasionally cause conflicts with existing builds of older versions.
-
-   a. Special `make` targets exist to help wipe the existing build of a
-      dependency. For example, `make -C deps clean-llvm` will clean out the
-      existing build of `llvm` so that `llvm` will be rebuilt from the
-      downloaded source distribution the next time `make` is called.
-      `make -C deps distclean-llvm` is a stronger wipe which will also delete
-      the downloaded source distribution, ensuring that a fresh copy of the
-      source distribution will be downloaded and that any new patches will be
-      applied the next time `make` is called.
-
-   b. To delete existing binaries of `julia` and all its dependencies,
-      delete the `./usr` directory _in the source tree_.
-
-3. If you've updated macOS recently, be sure to run `xcode-select --install` to update the command line tools.
-   Otherwise, you could run into errors for missing headers and libraries, such as
-   ```ld: library not found for -lcrt1.10.6.o```.
-
-4. If you've moved the source directory, you might get errors such as
-    ```CMake Error: The current CMakeCache.txt directory ... is different than the directory ... where     CMakeCache.txt was created.```, in which case you may delete the offending dependency under `deps`
-
-5. In extreme cases, you may wish to reset the source tree to a pristine state.
-   The following git commands may be helpful:
-
-   ```sh
-    git reset --hard #Forcibly remove any changes to any files under version control
-    git clean -x -f -d #Forcibly remove any file or directory not under version control
-   ```
-
-   _To avoid losing work, make sure you know what these commands do before you
-   run them. `git` will not be able to undo these changes!_
-
-
-
-### Uninstalling Julia
+If everything works correctly, you will see a Julia banner and an
+interactive prompt into which you can enter expressions for
+evaluation. (Errors related to libraries might be caused by old,
+incompatible libraries sitting around in your PATH. In this case, try
+moving the `julia` directory earlier in the PATH). Note that most of
+the instructions above apply to unix systems.
 
-Julia does not install anything outside the directory it was cloned into. Julia can be completely uninstalled by deleting this directory. Julia packages are installed in `~/.julia` by default, and can be uninstalled by deleting `~/.julia`.
+Your first test of Julia determines whether your build is working
+properly. From the UNIX/Windows command prompt inside the `julia`
+source directory, type `make testall`. You should see output that
+lists a series of running tests; if they complete without error, you
+should be in good shape to start using Julia.
 
-## Platform-Specific Build Notes
+You can read about [getting
+started](https://docs.julialang.org/en/stable/manual/getting-started/)
+in the manual.
 
-### Linux
+## Uninstalling Julia
```

Comment:
`## Uninstalling Julia`

Reply:
I intentionally made that change to have uninstall at the top level

Reply:
Why though? These instructions are for source  compilations, which is why it would make sense at level 3.